goodmap/core_api.py

```
@@ -137,4 +149,281 @@
             csrf_token = csrf_generator()
             return {"csrf_token": csrf_token}
 
+    @core_api.route("/admin/locations")
+    class AdminManageLocations(Resource):
+        def get(self):
+            """
+            Shows full list of locations, with optional server-side pagination, sorting, and filtering.
```

Ruff (0.11.9)

Line too long (103 > 100)

(E501)